site/en/docs/privacy-sandbox/glossary/index.md

```diff
@@ -108,11 +108,21 @@ out whether new data adds to entropy. For example, knowing a person is from
 Australia doesn't reduce entropy if you already know the person is from
 Kangaroo Island.
 
-## Federated identity (also known as federated login) {: #federated-identity }
+## Federated identity, federated login {: #federated-identity }
```

Comment:
```suggestion
## Federated identity (federated login) {: #federated-identity }
```

Reply:
How about this formatting if they are synonyms? Feels like it could be two things when it's comma separated. Just an idea, no strong opinion on this.

Reply:
I feel good about. I'm going to review the rest of the glossary to ensure it's the same everywhere.

Reply:
Hehe love having some good word nerd collaboration